CHANGELOG

```diff
@@ -5,6 +5,8 @@ Version 1.1.25 under development
 --------------------------------
 
 - Bug #4369: PHP 8.0 compatibility: Fix warning "Only the first byte will be assigned to the string offset" when generating code with Gii (marcovtwout)
+- Bug: createUpdateCommand does not accept just a table name when using MSSQL (c-schmitz)
```

Comment:
Please include the PR number #4374 and start the sentence with a capital
(or allow maintainer edits and I'll fix that :) )

Reply:
Fixed it :-)